pandas/tests/io/test_pytables.py

```diff
@@ -2192,9 +2192,9 @@ def test_unimplemented_dtypes_table_columns(self):
             pytest.raises(TypeError, store.append, 'df_unimplemented', df)
 
     @pytest.mark.skipif(
-        not _np_version_under1p15,
```

Comment:
don't define anything new here, juse use LooseVersion explicity to check for 1.15.0 as this is the only version affected.

Reply:
I have changed it. Could you please have a look at it @jreback. let me know if changes required.